.github/workflows/e2e.yml

```diff
@@ -126,6 +126,9 @@ jobs:
             - name: Print container logs
               if: ${{failure() || cancelled()}}
               run: docker compose -f docker-compose.test.yml --profile integration logs
+              env:
+                  INTEGRATION_TESTS_IMAGE: ${{steps.label.outputs.core_label}}
+                  JOBS_IMAGE: ${{steps.label.outputs.jobs_label}}
```

Comment:
it seemed like this step was often erroring in CI (though it didn't affect much since this only runs if there was already a failure). I think it just needed these env vars